content/articles/2021-11-october-updates.md

```diff
@@ -0,0 +1,40 @@
+Title: October updates
+Date: 2021-10-01 20:00 CET
+Tags: NumFOCUS,Updates
+slug: october-updates
+lang: en
+Author: Juan Luis Cano Rodríguez
+
+October was a particularly intense month, with lots of exciting news!
+
+We are thrilled to announce that NumFOCUS accepted to fund
+[our proposal to introduce array orbit types
+to accelerate parallel computing workflows in poliastro](https://github.com/poliastro/documents/blob/master/numfocus-sdg-2021-r3.md)
+as part of their Small Development Grants program.
+The proposal was written by Sebastian M. Ernst
+and we have already started coordinating what will be the next steps.
+
+In other news, a team of researchers participating in the
+[Global Trajectory Optimization Competition](https://sophia.estec.esa.int/gtoc_portal/)
+is using the poliastro,
+and they discovered a few bugs and made numerous code contributions.
+Special thanks to [Manuel López Ibáñez](https://github.com/MLopez-Ibanez)!
```

Comment:
@mlopez-ibanez are you OK with us disclosing this information? It's public for anybody that is curious enough to read the conversations in the repository, but wanted to ask anyway. This will be published in our public blog https://www.poliastro.space/blog/ and possibly in the upcoming NumFOCUS newsletter.

Reply:
I'm OK with it. Could you use http:/lopez-ibanez.eu instead ?

Reply:
I get a `ERR_TOO_MANY_REDIRECTS` from both Firefox and Chromium 🤔 in any case, I'll change the URL